src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    [CakeAliasCategory("ReportGenerator")]
+    public sealed class ReportGeneratorSettings
```

Comment:
@devlead @patriksvensson Should be Settings for a Tool include an optional ToolPath to allow the consumer to specify where the exe for a tool can be located?  I have done this on my PR's lately, but I have noticed that some others don't do this.  Is there a reason there?  Or what is the "norm".